src/olympia/files/utils.py

```diff
@@ -81,6 +81,12 @@ class ParseError(forms.ValidationError):
 _lock_count = {}
 
 
+# Acceptable extensions.
+# This is being used by `parse_addon` so please make sure we don't have
+# to touch add-ons before removing anything from this list.
+VALID_EXTENSIONS = ('.crx', '.xpi', '.jar', '.xml', '.json', '.zip')
```

Comment:
Should this be in base/constants.py?

Reply:
Now that you say that, I think it kinda makes sense. I'll move it there.

Reply:
Fied in 14a14bc500